### DISCUSSION TITLE
Any way of getting more charts for websocket

### DISCUSSION THREAD

**Discussion by aman-rajput40404** [artilleryio/artillery#1506] [Artillery]
Currently, the report for websocket only shows Error At Interval chart and Counter At Interval chart. Is there a way we get more charts which could answer the questions like:
How many concurrent users are active for how much of time and how many of them failed or completed?

I'm also not sure about the exact meaning of x axis and y axis. Does x axis shows v.users?
![image](https://user-images.githubusercontent.com/86342798/177899529-4d664e09-556a-4aa5-bb42-27342e1cfa4d.png)

**Comment by hassy** [comment]
hi @aman-rajput40404 

The X-axis in that chart is time. The purple line is showing a steady arrival rate of new virtual users. Something happens 150 seconds into the test which causes a number of them to fail, followed by a huge spike in failures between 170 and 180 seconds.
